README.md

```diff
@@ -21,6 +21,22 @@ pip install bettermdptools
 git clone https://github.com/jlm429/bettermdptools
 ```
 
+### Available Environments with Wrappers
+
+`bettermdptools` provides wrappers for both **grid-world** and **non-grid-world** environments. These wrappers enhance the functionality of the original environments, such as discretizing continuous state spaces. Some of these wrappers also allow adjustment of the state space size, such as in Cartpole.
+
+See the [`bettermdptools/bettermdptools/envs`](bettermdptools/bettermdptools/envs) directory for more details.
```

Comment:
Could you actually add a link to the documentation here? And add more description to the documentation if it isn't currently sufficient